mne_bids/datasets.py

```diff
@@ -10,7 +10,7 @@
 import os.path as op
 import shutil
 import tarfile
-import urllib
+import urllib.request
```

Comment:
why this?

Reply:
you can see the error in the AppVeyor test for #122 : https://ci.appveyor.com/project/jasmainak/mne-bids/builds/20571106
No idea why it is only happening with that PR as it doesn't touch the file in question, nor does it happen on the other CI's...

Reply:
@jasmainak yep, this line will fix that (as this is how I discovered the issue in the first place...)